### PR TITLE
internal: merge xds/envconfig into env package

### PR DESCRIPTION
**Pull request by dfawley**
Also contains #4922 

RELEASE NOTES: N/A

### CODE REVIEW

**Review by menghanl — APPROVED** [review]

**Comment by zasweq** [comment]
LGTM, but see my comment on #4922.

**Comment by dfawley** [comment]
PR updated with the changes made in #4922 to leave RBAC in place.

File: internal/xds/bootstrap.go [hunk 1]
```diff
@@ -79,11 +79,11 @@ func SetupBootstrapFile(opts BootstrapOptions) (func(), error) {
 	}
 	logger.Infof("Created bootstrap file at %q with contents: %s\n", f.Name(), bootstrapContents)
 
-	origBootstrapFileName := env.BootstrapFileName
-	env.BootstrapFileName = f.Name()
+	origBootstrapFileName := envconfig.BootstrapFileName
```
Comment (menghanl) [inline internal/xds/bootstrap.go:82]: Hmm, most (all?) of the xds variable names don't have "xds" in it. Would that be confusing?

**Review by menghanl — CHANGES_REQUESTED** [review]

File: internal/xds/bootstrap.go (hunk 1 above)
Comment (dfawley) [reply internal/xds/bootstrap.go]: Makes sense. I tweaked the variable names a bit; PTAL

**Review by menghanl — APPROVED** [review]

**Pull request merged**